app/src/main/java/com/sdp13epfl2021/projmag/activities/ProjectsListActivity.kt

```diff
@@ -20,6 +24,14 @@ class ProjectsListActivity : AppCompatActivity() {
         itemAdapter = ItemAdapter(this, myDataset)
         recyclerView.adapter = itemAdapter
         recyclerView.setHasFixedSize(true)
+
+        // get the fab and make it go to the Form activity
+        val fab: View = findViewById(R.id.fab)
+        fab.setOnClickListener {
```

Comment:
Maybe would it be cleaner to set the `onClickListener` in the XML or UI editor, with the `onClick` option